pd/src/genesis.rs

```diff
@@ -5,42 +5,34 @@ use rand_chacha::ChaCha20Rng;
 use serde::{Deserialize, Serialize};
 use serde_with::serde_as;
 
-use penumbra_crypto::{asset, Address, Fq, Note, Value};
+use penumbra_crypto::{note, Address, Fq, Note};
 
 pub fn generate_genesis_notes(
     rng: &mut ChaCha20Rng,
     genesis_allocations: Vec<GenesisAddr>,
-) -> GenesisNotes {
-    let mut notes = Vec::<Note>::new();
+) -> Vec<helpers::GenesisNote> {
+    let mut notes = Vec::<helpers::GenesisNote>::new();
     for genesis_addr in genesis_allocations {
-        let note = Note::new(
+        let note = helpers::GenesisNote::new(
             *genesis_addr.address.diversifier(),
             *genesis_addr.address.transmission_key(),
-            Value {
+            GenesisNoteValue {
                 amount: genesis_addr.amount,
-                asset_id: asset::Id::from(genesis_addr.denom.as_bytes()),
+                asset_denom: genesis_addr.denom,
             },
             Fq::rand(rng),
         )
         .expect("note created successfully");
         notes.push(note);
     }
-    GenesisNotes { notes }
+    notes
 }
 
-#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq)]
-#[serde(
-    try_from = "helpers::GenesisNotesHelper",
-    into = "helpers::GenesisNotesHelper"
-)]
-pub struct GenesisNotes {
-    notes: Vec<Note>,
-}
-
-impl GenesisNotes {
-    pub fn notes(&self) -> Vec<Note> {
-        self.notes.clone()
-    }
+#[derive(Clone, Debug, PartialEq, Eq)]
```

Comment:
OK I see that this `GenesisNotes`/`GenesisNote` refactor was done since we need to pass in the asset denomination in the genesis file instead of the asset ID to populate the asset registry :+1:. Do we want to preserve the `notes` key in the `app_state` field in the genesis file? Probably yes if we have other genesis application state, otherwise fine to remove. Do you know @hdevalence?

Reply:
Yeah, I think we should keep this filed under the `notes` key to leave open adding other data.